website/content/en/status/report-2025-04-2025-06/geomman.adoc

```diff
@@ -0,0 +1,26 @@
+=== Geomman Development
+
+Links: +
+link:https://wiki.freebsd.org/SummerOfCode2025Projects/FullDiskAdministrationToolForFreeBSD[Geomman GSoC wiki] +
+link:https://gitlab.com/brauliorivas/geomman[geomman Gitlab repository] +
+link:https://gitlab.com/alfix/bsddialog[bsddialog repository] +
+link:https://man.freebsd.org/cgi/man.cgi?query=sade&manpath=FreeBSD+14.3-RELEASE+and+Ports[sade]
+
+Contact: Braulio Rivas <brauliorivas@FreeBSD.org>
+
+Geomman is a new partition tool based on sade that brings more functionality as moving, copying and pasting partitions. Geomman is part of Google Summer of Code 2025. Currently, it is available on the Gitlab repository. But at some moment, it is expected to be a tool in the base system.
+
+Geomman is a TUI, thought to allow to grow, shrink, move, copy and paste partitions with filesystems other than UFS. For example, users may be able to create an exFAT partition, as well as to resize an ext4 filesystem. This would make partition management easier, because there are tools for each individual task (mainly depending on the filesystem), but no one that concentrates all cases in a single tool.
```

Comment:
"meant" or "designed" would work better than "thought" here, I believe.
s/allow to grow, shrink, move, copy and paste/allow growing, shrinking, moving, copying, and pasting/
s/no one/none/